src/wp-includes/js/customize-preview-widgets.js

```diff
@@ -286,7 +312,7 @@ wp.customize.widgetsPreview = wp.customize.WidgetCustomizerPreview = (function(
 		 *
 		 * @since 4.5.0
 		 *
-		 * @returns {Array}
+		 * @returns {Array} A shallow copy of the array containing widget IDs.
```

Comment:
You might consider adding `@throws` to this doc bloc:
![javascript_documentation_standards_ _make_wordpress_core](https://user-images.githubusercontent.com/325040/32053412-91881f36-ba5b-11e7-8949-575ad8a1bff1.png)

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/